.github/workflows/unittests.yml

```diff
@@ -64,90 +64,73 @@ jobs:
-          java -Xmx5G -XX:ErrorFile=/var/log/java/hs_err_pid%p.log -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc \
+          gluten-te/cbash.sh \
+          "cd /opt/ \
+          && git clone -b main https://github.com/zhztheplayer/gluten-it.git gluten-it \
+          && cd gluten-it \
+          && mvn clean package -Pspark-3.2 -Pgluten-velox -Darrow.version=10.0.0-SNAPSHOT \
+          && java -Xmx5G -XX:ErrorFile=/var/log/java/hs_err_pid%p.log -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc \
             --backend-type=velox --benchmark-type=h --fixed-width-as-double --disable-aqe --off-heap-size=10g -s=1.0 --cpus=4 --iterations=1 \
-            --queries=q1,q2,q3,q4,q5,q6,q7,q8,q9,q10,q11,q12,q13,q14,q15,q16,q17,q18,q19,q20,q21,q22
+            --queries=q1,q2,q3,q4,q5,q6,q7,q8,q9,q10,q11,q12,q13,q14,q15,q16,q17,q18,q19,q20,q21,q22"
 
       - name: TPC-H Parquet SF1.0 local spark3.3
+        env:
+          HTTP_PROXY_HOST: ${{ secrets.SR404_PROXY_HOST }}
+          HTTP_PROXY_PORT: ${{ secrets.SR404_PROXY_PORT }}
         run: |
-          cd gluten-it
-          mvn clean package -Pgluten-velox -Darrow.version=10.0.0-SNAPSHOT -Pspark-3.3
-          java -Xmx5G -XX:ErrorFile=/var/log/java/hs_err_pid%p.log -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc \
+          gluten-te/cbash.sh \
+          "cd /opt/ \
+          && git clone -b main https://github.com/zhztheplayer/gluten-it.git gluten-it \
+          && cd gluten-it \
+          && mvn clean package -Pgluten-velox -Darrow.version=10.0.0-SNAPSHOT -Pspark-3.3 \
+          && java -Xmx5G -XX:ErrorFile=/var/log/java/hs_err_pid%p.log -cp target/gluten-it-1.0-SNAPSHOT-jar-with-dependencies.jar io.glutenproject.integration.tpc.Tpc \
             --backend-type=velox --benchmark-type=h --fixed-width-as-double --disable-aqe --off-heap-size=10g -s=1.0 --cpus=4 --iterations=1 \
-            --queries=q1,q2,q3,q4,q5,q6,q7,q8,q9,q10,q11,q12,q13,q14,q15,q16,q17,q18,q19,q20,q21,q22
+            --queries=q1,q2,q3,q4,q5,q6,q7,q8,q9,q10,q11,q12,q13,q14,q15,q16,q17,q18,q19,q20,q21,q22"
 
       - name: Build spark3.2 and spark3.3 at the same time
         run: |
-          mvn clean install -Pspark-3.2 -Pspark-3.3 -DskipTests -Dcheckstyle.skip -Dscalastyle.skip=true -Pbackends-velox 
+          mvn clean install -Pspark-3.2 -Pspark-3.3 -DskipTests -Dcheckstyle.skip -Dscalastyle.skip=true -Pbackends-velox
+
+      # Deprecated
+      #- name: Run micro benchmarks
+      #  run: |
+      #    cd cpp/velox/benchmarks/
+      #    ./generic_benchmark \
+      #    hash_join.json \
+      #    parquet/bm_lineitem/part-00000-8bd1ea02-5f13-449f-b7ef-e32a0f11583d-c000.snappy.parquet \
+      #    parquet/bm_part/part-00000-d8bbcbeb-f056-4b7f-8f80-7e5ee7260b9f-c000.snappy.parquet
```

Comment:
Should we remove this?

Reply:
After substrait change to version 7, all the plan.json is out of date, we can keep it as remark until some one fixes it